lib/src/main/kotlin/com/lemonappdev/konsist/api/ext/list/KoAnnotationProviderListExt.kt

```diff
@@ -78,56 +78,56 @@ fun <T : KoAnnotationProvider> List<T>.withoutSomeAnnotations(
 /**
  * List containing elements with all annotations of type.
  *
- * @param annotation The Kotlin class representing annotation to include.
- * @param annotations The Kotlin classes representing annotations to include.
+ * @param kClass The Kotlin class representing annotation to include.
+ * @param kClasses The Kotlin classes representing annotations to include.
  * @return A list containing elements with all the specified annotations.
  */
 fun <T : KoAnnotationProvider> List<T>.withAllAnnotationsOf(
-    annotation: KClass<*>,
```

Comment:
I don't know if changing like this `annotation: KClass<*>` to `kClass: KClass<*>` are ok.
Advantages: Developer knows that he must pass annotation class with the `::class` suffix.
Disadvantages: It doesn't describe that developer can pass directly annotation classes (such as  `Suppress::class etc)

Reply:
It's fine